### DISCUSSION TITLE
C++ PDK?

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [extism/extism#441] [Q&A]
I want to have a near-seamless integration with a C++ game engine so that it can run almost-native extensions rather than using a wacky scripting language called Pawn. While there is a C PDK I'm very curious why not so for the C++.

**Comment by stevefan1999-personal** [comment]
I don't want to use the Host SDK because people can't directly port their existing C++ logic to Extism. Meanwhile having a SWIG to generate a custom C wrapper may work but this would get ugly pretty fast...

**Comment by zshipko** [comment]
A C++ PDK is definitely something we would like to have but just haven't gotten around to making it yet!
